sdk/src/main/java/com/mapbox/maps/renderer/MapboxRenderThread.kt

```diff
@@ -176,20 +193,33 @@ internal class MapboxRenderThread : Choreographer.FrameCallback {
     if (expectedVsyncWakeTimeNs > currentTimeNs) {
       return
     }
+
+    if (widgetRenderer.needRender) {
+      eglCore.makeNothingCurrent()
+      widgetRenderer.updateTexture()
+      eglCore.makeCurrent(eglSurface)
+    }
     mapboxRenderer.onDrawFrame()
-    // snapshots should be taken before swapBuffers otherwise buffers may be already empty
+
+    if (widgetRenderer.getTextureId() != 0) {
+      // needed to draw widgets on top of 3d location puck
+      GLES20.glDisable(GLES20.GL_STENCIL_TEST)
```

Comment:
if we know it was enabled - shouldn't we explicitly enable it after rendering a widget?

Reply:
Removed - not needed with the new GL native anymore